pallets/cosmos-abci/abci/src/grpc.rs

```diff
@@ -45,9 +47,18 @@ impl crate::AbciInterface for AbciinterfaceGrpc {
     fn check_tx(
         &mut self,
         tx: Vec<u8>,
-        r#type: i32,
+        tx_type: i32,
```

Comment:
As you explicitly check that transaction has been proceed before, we do not need this flag as an argument here.